src/enhanced-textarea.jsx

```diff
@@ -73,8 +77,17 @@ const EnhancedTextarea = React.createClass({
     };
   },
 
+  handleResize(e) {
+    this._syncHeightWithShadow(undefined, e);
+  },
+
   componentDidMount() {
     this._syncHeightWithShadow();
+    window.addEventListener('resize', this.handleResize);
+  },
+
+  componentWillUnmount: function() {
+    window.removeEventListener('resize', this.handleResize);
```

Comment:
You know how much I love how `react-event-listener` works, I've bragged about it before :laughing: So yeah absolutely :+1: :+1: